### PR TITLE
Fix: Inconsistent capitalization in product

### PR DESCRIPTION
**Pull request by dukenv0307**
### Details

Inconsistent capitalization in product
### Fixed Issues

$ https://github.com/Expensify/App/issues/17239
PROPOSAL: https://github.com/Expensify/App/issues/17239#issuecomment-1503682816

### Tests

Note: Case 1 in test could be only tested in desktop web and desktop app 

- The 'C' should be small in the tooltip for the call icon
1. Login with any account
2. Go to any chat
3. Hover call icon 
4. Verify that the content of tooltip is "Start a call"

- Text in Transfer Balance modal should be consistent capitalization and the header in Choose Transfer Account modal "A" should be in small letters
1. Login with any account
2. Go to Setting > Payments > Transfer balance if account has balance or go to url '/settings/payments/transfer-balance' if else
3. Verify that the header in this modal is 'Transfer balance'
4. Verify that option 1 is "Instant (Debit card)"
5. Verify that option 2 is "1-3 Business days (Bank account)"
6. Verify that the text that below two option is "Which account"
7. Select any option to go to Choose Transfer Account modal
8. Verify that the header in this modal is 'Choose account'

- "Terms of Service" and "Privacy" text should be consistent in product
1. Go to login page
2. Enter the email of your account and click continue
3. In the page to enter the password, verify that the term text is "By logging in, you agree to the Terms of Service and Privacy."
4. Enter your password to login
5. After login successfully, go to Setting > About
9. Verify that the content in footer is "Read the Terms of Service and Privacy"
10. Switch to Spanish language
11. Go to Setting > Workspaces > Connect bank account
12. In step 1, doesn't check to checkbox and click Continue button
13. Verify that error message "Debes aceptar los Términos de Servicio para continuar" is displayed below checkbox 

- The header in the Delete Account modal "A" should be in small letters
1. Login with any account
2. Go to Setting > Payments
3. Select a payment method to delete
4. Verify that the header in the Delete account modal is "Delete account"

- Uploading Attachment... message "A" should be in small letters
1. Login with any account
2. Go to any chat
3. Upload a attachment
4. Verify that Uploading attachment message is 'Uploading attachment'

- Error in add bank account form should be consistent capitalization
1. Login with any account 
2. Open Setting > Workspaces
3. Open any workspaces
4. Open connect bank account
5. Click on connect manually
6. In step 1, without filling any detail, click on continue
7. Verify that the error message of Routing number field is "Please enter a valid routing number"
8. Verify that the error message of Account number is "Please enter a valid account number"
9. Fill in detail in step one with valid value, accept terms and click continue to go to step 2 (i.e routin no = 123456780, account no = 123456789)
10. In step 2, without filling any detail, click on save & continue
11. Verify that the error message of Tax ID number field is "Please enter a valid tax ID number"
12. Verify that the error message of Company type is "Please enter a valid company type"
13. Verify that the error message of Company address is "Please enter a valid street address that is not a PO box"

- [x] Verify that no errors appear in the JS console

### Offline tests
Same above

### QA Steps

Note: Case 1 in test could be only tested in desktop web and desktop app 

- The 'C' should be small in the tooltip for the call icon
1. Login with any account
2. Go to any chat
3. Hover call icon 
4. Verify that the content of tooltip is "Start a call"

- Text in Transfer Balance modal should be consistent capitalization and the header in Choose Transfer Account modal "A" should be in small letters
1. Login with any account
2. Go to Setting > Payments > Transfer balance if account has balance or go to url '/settings/payments/transfer-balance' if else
3. Verify that the header in this modal is 'Transfer balance'
4. Verify that option 1 is "Instant (Debit card)"
5. Verify that option 2 is "1-3 Business days (Bank account)"
6. Verify that the text that below two option is "Which account"
7. Select any option to go to Choose Transfer Account modal
8. Verify that the header in this modal is 'Choose account'

- "Terms of Service" and "Privacy" text should be consistent in product
1. Go to login page
2. Enter the email of your account and click continue
3. In the page to enter the password, verify that the term text is "By logging in, you agree to the Terms of Service and Privacy."
4. Enter your password to login
5. After login successfully, go to Setting > About
9. Verify that the content in footer is "Read the Terms of Service and Privacy"
10. Switch to Spanish language
11. Go to Setting > Workspaces > Connect bank account
12. In step 1, doesn't check to checkbox and click Continue button
13. Verify that error message "Debes aceptar los Términos de Servicio para continuar" is displayed below checkbox 

- The header in the Delete Account modal "A" should be in small letters
1. Login with any account
2. Go to Setting > Payments
3. Select a payment method to delete
4. Verify that the header in the Delete account modal is "Delete account"

- Uploading Attachment... message "A" should be in small letters
1. Login with any account
2. Go to any chat
3. Upload a attachment
4. Verify that Uploading attachment message is 'Uploading attachment'

- Error in add bank account form should be consistent capitalization
1. Login with any account 
2. Open Setting > Workspaces
3. Open any workspaces
4. Open connect bank account
5. Click on connect manually
6. In step 1, without filling any detail, click on continue
7. Verify that the error message of Routing number field is "Please enter a valid routing number"
8. Verify that the error message of Account number is "Please enter a valid account number"
9. Fill in detail in step one with valid value, accept terms and click continue to go to step 2 (i.e routin no = 123456780, account no = 123456789)
10. In step 2, without filling any detail, click on save & continue
11. Verify that the error message of Tax ID number field is "Please enter a valid tax ID number"
12. Verify that the error message of Company type is "Please enter a valid company type"
13. Verify that the error message of Company address is "Please enter a valid street address that is not a PO box"
- [ ] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
-  [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

- The 'C' should be small in the tooltip for the call icon

Screenshot from 2023-04-13 11-17-44

- Text in Transfer Balance modal should be consistent capitalization and the header in Choose Transfer Account modal "A" should be in small letters

Screenshot from 2023-04-19 13-15-26

Screenshot from 2023-04-13 11-18-14

- "Terms of Service" and "Privacy" text should be consistent in product

Screenshot 2023-04-19 20:33:47

Screenshot from 2023-04-19 13-16-53

Screenshot from 2023-04-13 17-16-49

Screenshot 2023-04-19 20:43:44

- The header in the Delete Account modal "A" should be in small letters

Screenshot from 2023-04-13 11-18-44

- Uploading Attachment... message "A" should be in small letters

Screenshot from 2023-04-13 11-19-06

- Error in add bank account form should be consistent capitalization

Screenshot from 2023-04-19 13-17-32

Screenshot from 2023-04-19 13-19-05

</details>

<details>
<summary>Mobile Web - Chrome</summary>

- Text in Transfer Balance modal should be consistent capitalization and the header in Choose Transfer Account modal "A" should be in small letters

- "Terms of Service" and "Privacy" text should be consistent in product

- The header in the Delete Account modal "A" should be in small letters

- Uploading Attachment... message "A" should be in small letters

 
- Error in add bank account form should be consistent capitalization

</details>

<details>
<summary>Mobile Web - Safari</summary>

- Text in Transfer Balance modal should be consistent capitalization and the header in Choose Transfer Account modal "A" should be in small letters
Screenshot 2023-04-13 at 17 40 44
Screenshot 2023-04-19 at 14 28 05

- "Terms of Service" text should be consistent in product
Screenshot 2023-04-19 at 14 28 34
Screenshot 2023-04-19 at 20 52 59
Screenshot 2023-04-19 at 20 52 15

Screenshot 2023-04-13 at 17 41 58

- The header in the Delete Account modal "A" should be in small letters
Screenshot 2023-04-13 at 17 47 02

- Uploading Attachment... message "A" should be in small letters
Screenshot 2023-04-13 at 17 47 52

- Error in add bank account form should be consistent capitalization
Screenshot 2023-04-19 at 14 31 23
Screenshot 2023-04-19 at 14 31 38

 
</details>

<details>
<summary>Desktop</summary>

- The 'C' should be small in the tooltip for the call icon
Screenshot 2023-04-13 at 17 54 11

- Text in Transfer Balance modal should be consistent capitalization and the header in Choose Transfer Account modal "A" should be in small letters

Screenshot 2023-04-19 at 14 38 10

 Screenshot 2023-04-13 at 17 55 15 

- "Terms of Service" and "Privacy" text should be consistent in product

Screenshot 2023-04-19 at 14 38 26

Screenshot 2023-04-19 at 20 55 38
Screenshot 2023-04-19 at 20 55 54

Screenshot 2023-04-13 at 17 57 08 

- The header in the Delete Account modal "A" should be in small letters
Screenshot 2023-04-13 at 17 59 03

- Uploading Attachment... message "A" should be in small letters
Screenshot 2023-04-13 at 17 59 38

- Error in add bank account form should be consistent capitalization
Screenshot 2023-04-19 at 14 39 33

Screenshot 2023-04-19 at 14 39 40

</details>

<details>
<summary>iOS</summary>

- Text in Transfer Balance modal should be consistent capitalization and the header in Choose Transfer Account modal "A" should be in small letters
Screenshot 2023-04-19 at 14 54 14

Screenshot 2023-04-13 at 17 49 33

- "Terms of Service" and "Privacy" text should be consistent in product
Screenshot 2023-04-19 at 14 54 21

Screenshot 2023-04-19 at 21 06 36
Screenshot 2023-04-19 at 21 06 57

Screenshot 2023-04-13 at 17 50 46

- The header in the Delete Account modal "A" should be in small letters
Screenshot 2023-04-13 at 17 51 15

- Uploading Attachment... message "A" should be in small letters
Screenshot 2023-04-13 at 17 51 45

- Error in add bank account form should be consistent capitalization

Screenshot 2023-04-19 at 14 55 12

Screenshot 2023-04-19 at 14 55 05

</details>

<details>
<summary>Android</summary>

- Text in Transfer Balance modal should be consistent capitalization and the header in Choose Transfer Account modal "A" should be in small letters

Screenshot_1681888165

image_2023_04_13T09_03_39_201Z

- "Terms of Service" and "Privacy" text should be consistent in product

Screenshot_1681911632

Screenshot_1681886818

image_2023_04_13T09_18_21_294Z

Screenshot_1681911879

- The header in the Delete Account modal "A" should be in small letters

image_2023_04_13T09_09_02_946Z

- Uploading Attachment... message "A" should be in small letters

image_2023_04_13T09_33_15_043Z

- Error in add bank account form should be consistent capitalization

Screenshot_1681888184

Screenshot_1681888204

Screenshot_1681888206

 
</details>

### CODE REVIEW

**Comment by dukenv0307** [comment]
@0xmiroslav Thanks for reminding me. Noted

File: src/libs/ReportUtils.js
Comment (thienlnam) [reply src/libs/ReportUtils.js:934]: Nice catch, yeah I'm not sure why this copy was not ever localized.  Agree we can just leave it since we're going to be removing it soon

**Comment by dukenv0307** [comment]
@0xmiroslav  @thienlnam  Is there any other problem in this PR?

**Comment by thienlnam** [comment]
Sasha left some confirmation - let's also make sure to address the cases in the latest merged issue, this should address all the inconsistent capitalization in product

**Comment by dukenv0307** [comment]
@thienlnam @0xmiroslav I just updated the PR, please help to check again and I will update screenshots after your confirm

**Comment by dukenv0307** [comment]
@0xmiroslav I just updated all screenshot, help to check again

**Comment by dukenv0307** [comment]
@0xmiroslav Spanish changes only in case "Terms of Service" and "Privacy" text should be consistent in product and you can see it in all platform

**Comment by dukenv0307** [comment]
@0xmiroslav Login page changed in main but this text is same. Do I need to update screenshots in login page

**Comment by dukenv0307** [comment]
@0xmiroslav sorry I was busy earlier, I will update it right now

**Comment by dukenv0307** [comment]
@0xmiroslav I just updated. Help to check again

**Review by 0xmiroslav — DISMISSED** [review]
2 minor concerns:
- Inconsistency - (Debit card), (Bank account) is missing in Spanish. This is not capitalization issue so out of scope.
Screenshot 2023-04-19 at 4 41 35 PM
Screenshot 2023-04-19 at 4 42 15 PM


- `T`erms of `S`ervice and `P`rivacy capitalization together in a sentence (already [raised](https://github.com/Expensify/App/issues/17239#issuecomment-1514107915) in linked issue but got no answer yet)
Screenshot 2023-04-19 at 4 44 31 PM
Screenshot 2023-04-19 at 4 44 38 PM


Other than that, looks good.
cc: @thienlnam

**Review by thienlnam — DISMISSED** [review]
Looks good! Thanks for the review, going to wait for confirmation but I agree with your stance on the capitalization

**Comment by dukenv0307** [comment]
@0xmiroslav I just fixed, please check again

**Review by 0xmiroslav — APPROVED** [review]
Thanks for the quick fix. We're still waiting for "Terms of Service and Privacy" confirmation before final merge.

**Review by thienlnam — APPROVED** [review]
Cool we're good to go here! Nice work everyone

**Pull request merged**
